protected/humhub/modules/user/components/User.php

```diff
@@ -147,6 +147,17 @@ public function afterLogin($identity, $cookieBased, $duration)
         parent::afterLogin($identity, $cookieBased, $duration);
     }
 
+    /**
+     * Set updated_by for current user
+     *
+     * @param $identity
+     * @return void
+     */
+    public function selfSetUpdatedBy($identity)
```

Comment:
I would like to keep this component as compact as possible. And prefer instead, two private methods:

- tour\Module::getIsFirstLogin(User $user): bool
- tour\Module::setFirstLoginDone(User $user): void